src/tests/GfxBatchRendererTest.cpp

```diff
@@ -1710,6 +1713,27 @@ void GfxBatchRendererTest::multipleMeshes() {
   CORRADE_COMPARE(color.pixels<Mn::Color4ub>()[24][88], 0xcc00cc_rgb);
 }
 
+void GfxBatchRendererTest::emptyScene() {
+  // clang-format off
+  esp::gfx_batch::RendererStandalone renderer{
+      esp::gfx_batch::RendererConfiguration{}
+          .setTileSizeCount({64, 48}, {2, 2}),
+      esp::gfx_batch::RendererStandaloneConfiguration{}
+          .setFlags(esp::gfx_batch::RendererStandaloneFlag::QuietLog)
+  };
+  // clang-format on
+
+  for (int i = 0; i < 4; ++i) {
+    esp::gfx_batch::SceneStats stats = renderer.sceneStats(i);
+    CORRADE_COMPARE(stats.nodeCount, 0);
+    CORRADE_COMPARE(stats.drawCount, 0);
+    CORRADE_COMPARE(stats.drawBatchCount, 0);
+  }
```

Comment:
No need to test this here, checking stats before any file is added is already done elsewhere.